CHANGELOG.md

```diff
@@ -2,13 +2,181 @@
 
 ## 10.0.0
 
-Teleport 10.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
+Teleport 10 is a major release that brings:
 
-### Breaking Changes
+* Passwordless (Preview)
+* Resource Access Requests (Preview)
+* Proxy Peering (Preview)
+* IP-Based Restrictions (Preview)
+* Automatic User Provisioning (Preview)
+* Audit Logging for Microsoft SQL Server Database Access
+* Snowflake Database Access (Preview)
+* ElastiCache/MemoryDB Database Access (Preview)
+* Teleport Connect for Server and Database Access (Preview)
+* Machine ID Database Access Support (Preview)
+
+### Passwordless (Preview)
+
+Teleport 10 introduces passwordless support to your clusters. Any cluster with
+WebAuthn configured automatically benefits from passwordless. To use
```

Comment:
> Any cluster with WebAuthn configured automatically benefits from passwordless.

I don't actually think this is true because admins have to turn on passwordless as an authentication method.